app/controllers/dashboards_controller.rb

```diff
@@ -1,11 +1,13 @@
-# typed: true
+# typed: false
 # frozen_string_literal: true
 
 # Displays the list of collections to the user
 class DashboardsController < ApplicationController
   before_action :authenticate_user!
+  verify_authorized
 
   def show
     @collections = authorized_scope(Collection.all)
+    authorize! :dashboard
```

Comment:
I don't think it's a big deal, but I wonder if we need line 11 below line 10 here. (Not familiar with authorized_scope.)

Reply:
i'm also unfamiliar with `authorized_scope`, and hadn't really considered the possibility it'd cover the authorization (didn't run the tests before adding the `authorize!` on line 11.

but i just tried out commenting out line 11, and apparently `authorized_scope` doesn't fulfill the `verify_authorized` requirement, because leaving off `authorized! :dashboard` gets me 6 failures along the lines of:

```
  1) Create a new collection and deposit to it when successful deposit deposits and renders work show page
     Failure/Error: @app.call(env)
     
     ActionPolicy::UnauthorizedAction:
       Action 'dashboards#show' hasn't been authorized
     # ./lib/test_shibboleth_headers.rb:17:in `call'
     # ------------------
     # --- Caused by: ---
     # Capybara::ElementNotFound:
     #   Unable to find link "+ Create a new collection"
     #   ./spec/features/create_new_work_spec.rb:20:in `block (3 levels) in <top (required)>'
```

(the other 5 were were in `dashboards_spec.rb`)

Reply:
Ah, I should have been more specific: I was wondering if these two lines could switch places.

Reply:
ah, gotcha, fixed in #277